extensions/morning-coffee/src/components/actions/website-delete.tsx

```diff
@@ -0,0 +1,13 @@
+import { Action, Icon } from "@raycast/api";
+
+interface ActionWebsiteDeleteInterface {
+  onDelete(): void;
+}
+
+export const ActionWebsiteDelete = ({ onDelete }: ActionWebsiteDeleteInterface) => {
+  return (
+    <Action icon={Icon.Trash} title="Delete Website" shortcut={{ modifiers: ["ctrl"], key: "x" }} onAction={onDelete} />
+  );
```

Comment:
```suggestion
return (
    <Action
      icon={Icon.Trash}
      title="Delete Website"
      style={Action.Style.Destructive}
      shortcut={{ modifiers: ["ctrl"], key: "x" }}
      onAction={onDelete}
    />
  );
```
Maybe also add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert)

Reply:
Added destructive styles, looks great. It also uses "confirmAlert" now.